packages/manager/modules/catalog-price/src/catalog-price.controller.js

```diff
@@ -38,16 +42,66 @@ export default class {
     this.ovhSubsidiary = get(this.user, 'ovhSubsidiary', '');
   }
 
+  /**
+   * Transform the price in uCents to a human readable and regionalized text based values
+   * @param priceInCents the price to format
+   * @returns {string} the formatted and regionalized prices, appended with currency unit.
+   */
   getPriceText(priceInCents) {
     const locale = this.TranslateService.getUserLocale().replace('_', '-');
-    const price = this.getIntervalPrice(priceInCents / 100000000);
+    let price;
+    if (this.performRounding) {
+      price = this.getIntervalPrice(priceInCents / 100000000);
+    } else {
+      price = this.getIntervalPriceWithoutRounding(priceInCents / 100000000);
+    }
```

Comment:
**suggestion:** for simplicty
```suggestion
    let price = priceInCents / 100000000;
    price = this.performRounding ? this.getIntervalPrice(price) : this.getIntervalPriceWithoutRounding(price)
```

Reply:
Hi ! Will apply while separating the comits :)